DataFormats/VertexSoA/src/classes_def.xml

```diff
@@ -1,8 +1,8 @@
 <lcgdict>
   <class name="reco::ZVertexSoA"/>
   <class name="PortableHostCollection<reco::ZVertexSoA>"/>
-  <class name="ZVertexHost" ClassVersion="3">
-    <version ClassVersion="3" checksum="1989784241"/>
+  <class name="ZVertexHost" ClassVersion="4">
+    <version ClassVersion="4" checksum="1552837222"/>
```

Comment:
Given that `ZVertexHost` is a template instantiation, the class version should be removed. (I might have given false advice in the earlier PR)
```suggestion
  <class name="ZVertexHost"/>
```

For class templates the class version should be defined in the class definition with `CMS_CLASS_VERSION()` macro, but given that the schema evolution of these things is really unexplored territory, maybe that's best to be left to be figured out a little bit later?